nca/nca_cli.py

```diff
@@ -324,11 +326,17 @@ def nca_main(argv=None):
     if args.daemon:
         return RestServer(args.ns_list, args.pod_list, args.resource_list).run()
 
-    if args.period <= 0:
-        ret_val = run_args(args)
-        return 0 if args.return_0 else ret_val
-
-    _do_every(args.period * 60, run_args, args)
+    try:
+        if args.period <= 0:
+            ret_val = run_args(args)
+            return 0 if args.return_0 else ret_val
+
+        _do_every(args.period * 60, run_args, args)
+    except Exception as e:
+        print('Error: {}'.format(e))
```

Comment:
```suggestion
        print(f'Error: {e}')
```
We always use f-strings in this project

Reply:
done